_pages/pt/developer-engagement.md

```diff
@@ -9,11 +9,11 @@ ref: developer-engagement
 {: .right}
 ![image](/asserts/img/development.png)
 
-Developers are very important for Mozilla. Everything we do is not only to serve our hundreds of millions of users, but to bring Mozilla's manifesto and values to web developers and make their lives easier. We provide many things for developers. To name just a few, we have first class developer tools in Firefox, a world leader add-ons platform, and widely recognised developer documentation portal in MDN.
+Desenvolvedores são muito importantes para Mozilla. Tudo que nós fazemos não é para somente servir nossas centenas de milhares de usuários, mas para trazer o manifesto da Mozilla e valores para desenvolvedores web e fazer suas vidas mais fácil. Nós provemos muitas coisas para desenvolvedores. Para citar apenas algumas, nós temos a primeira ferramenta de desenvolvimento em classe no Firefox, um plataforma de liderança global de add-ons, e o amplamente reconhecido portal de documentação para desenvolvedores o MDN.
 
-Because of this, we always aim to work more with developers to hear what their needs are and to bring our stories to them. This is where you come in. These activities are about working with developers and telling them about the latest and greatest that is happening with Mozilla and around the Web.
+Por conta disso, nós sempre com o objetivo de trabalhar mais com os desenvolvedores e para ouvir quais são suas necessidades e trazer nossas histórias para eles. É ai onde você entra. Essas atividades são sobre como trabalhar com os desenvolvedores e dizer-lhes sobre o que há de maior e mais recente que está acontecendo com a Mozilla e em torno da Web.
```

Comment:
"Por conta disso, nós sempre almejamos trabalhar mais com desenvolvedores para ouvir quais são suas necessidades e trazer nossas histórias a eles."

"Essas atividades são sobre como trabalhar com os desenvolvedores e contar-lhes sobre o que há de melhor e mais recente que está acontecendo com a Mozilla e pela Web." - just update a few words here.

I got a little confused reading the first phrase. I translate it from the beginning.